source/features/clean-repo-sidebar.tsx

```diff
@@ -25,14 +25,6 @@ async function cleanReleases(): Promise<void> {
 		.previousElementSibling! // About’s .BorderGrid-row
 		.firstElementChild! // About’s .BorderGrid-cell
 		.classList.add('border-0', 'pb-0');
-
-	// Align latest tag icon with the icons of other meta links
-	const tagIcon = select('.octicon-tag:not(.color-fg-success)', releasesSection)!;
-	if (tagIcon) {
-		tagIcon.classList.add('mr-2');
-		// Remove whitespace node
-		tagIcon.nextSibling!.remove();
-	}
```

Comment:
This doesn't seem to be hit anymore. The only tag icon I see is green.

<img width="206" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/199896168-f7a93063-3c54-454c-9254-daa13c0f458e.png">


cc @cheap-glitch from https://github.com/refined-github/refined-github/pull/3974